components/concept/toolbar/src/main/java/mozilla/components/concept/toolbar/Toolbar.kt

```diff
@@ -16,6 +16,13 @@ interface Toolbar {
      */
     fun displayUrl(url: String)
 
+    /**
+     * Displays or updates the progress made loading the current URL.
+     *
+     * @param progress the progress made as a value between 0 and 100
+     */
+    fun displayProgress(progress: Int)
+
     /**
```

Comment:
Let's not add the Toolbar code just yet. I'm currently refactoring / building the toolbar and this will cause a lot of merge conflicts.. :)

Reply:
OK, dropping the toolbar code from this commit.